Mage/src/main/java/mage/abilities/effects/ContinuousEffectImpl.java

```diff
@@ -93,7 +95,7 @@ protected ContinuousEffectImpl(final ContinuousEffectImpl effect) {
         this.startingControllerId = effect.startingControllerId;
         this.startingTurnWasActive = effect.startingTurnWasActive;
         this.effectStartingOnTurn = effect.effectStartingOnTurn;
-        this.effectStartingEndStep = effect.effectStartingEndStep;
+        this.effectControllerStartingEndStep = effect.effectControllerStartingEndStep;
```

Comment:
Miss copy constructor fields — added 3 new, but copy changed for 1:

![IMG_0869](https://github.com/user-attachments/assets/95f96592-3c5e-4771-9bd2-6d8927a60286)